### PR TITLE
Services Card Improved

### PR DESCRIPTION
**Pull request by rojandahal**
-Import svg 
-Improve Navbar

### CODE REVIEW

File: src/assets/pdf/data.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        {
+        "url": "/blog",
+        "title": "Services"
+        }
+    ],
+    "services": [
+        {
+        "title": "Web Design",
+        "description": "My web design service combines creativity, usability, and branding to create custom websites that captivate users, meet your business needs, and achieve your online goals.",
+        "icon": "a"
```
Comment (aayussraut) [inline src/assets/pdf/data.json]: icon ko naam change gara ali understandable rakha

File: src/components/Slider/Skills.js [hunk 2]
```diff
@@ -10,10 +10,10 @@ export default function Skills() {
...
     <>
       <div className="skills-card-container">
-        <Card image={imgJs} cardClass="card-skills" title={"JavaScript"}></Card>
-        <Card image={imgReact} cardClass="card-skills" title={"React"}></Card>
-        <Card image={imgJava} cardClass="card-skills" title={"Java"}></Card>
-        <Card image={imgNode} cardClass="card-skills" title={"Node"}></Card>
+        <Card icon={imgJs} cardClass="card-skills" title={"JavaScript"}></Card>
+        <Card icon={imgReact} cardClass="card-skills" title={"React"}></Card>
+        <Card icon={imgJava} cardClass="card-skills" title={"Java"}></Card>
+        <Card icon={imgNode} cardClass="card-skills" title={"Node"}></Card>
```
Comment (aayussraut) [inline src/components/Slider/Skills.js]: yeslai ne map gardeu

**Review by aayussraut — COMMENTED** [review]
- Change gara ani merge gardeu

**Comment by rojandahal** [comment]
-Merge Conflict Resolved

**Pull request merged**